src/api/merchants/merchant-configs.md

```diff
@@ -55,6 +55,7 @@ A Merchant Config defines the available payment options for paying a [Payment Re
 | walletId           | String | Merchant's Centrapay Settlement Wallet to receive payments. Optional for `centrapay.nzd` types, if not supplied then a settlement wallet may be created and then assigned. |
 | terminalId         | String | Merchant's Epay terminal id.                                                                                                                                               |
 | wavesPublicAddress | String | Merchant's public waves address. Required for `zap.main` types.                                                                                                            |
+| externalMerchantId | String | Id of a merchant in an external system. Required for `paypal.usd` types.                                                                                                            |
```

Comment:
PayPal's capture requests also need a store ID - do we need a field on the merchant config for this too?

Reply:
Yep, changed this to include two new fields called paypalMerchantAccountId and paypalStoreId instead.